backend/handlers.go

```diff
@@ -38,6 +38,16 @@ func (cs *server) cssHandler(w http.ResponseWriter, r *http.Request, _ httproute
 	}
 }
 
+//Helper function which splits into two parts the sensorID by the hyphen
+func splitSensorID(sensorID string) string {
```

Comment:
Instead of assuming a UUID, we can also take the first 8 characters. 
Maybe also add a check that there are at least 8 characters.